test/clustertest/tests/MultipleLeaderSyncTest.cpp

```diff
@@ -0,0 +1,91 @@
+#include "../BedrockClusterTester.h"
+
+struct MultipleLeaderSyncTest : tpunit::TestFixture {
+    MultipleLeaderSyncTest()
+        : tpunit::TestFixture("MultipleLeaderSyncTest",
+                              BEFORE_CLASS(MultipleLeaderSyncTest::setup),
+                              AFTER_CLASS(MultipleLeaderSyncTest::teardown),
+                              TEST(MultipleLeaderSyncTest::test)
+                             ) { }
+
+    BedrockClusterTester* tester;
+
+    void setup() {
+        // create a 5 node cluster
+        tester = new BedrockClusterTester(BedrockClusterTester::FIVE_NODE_CLUSTER, {"CREATE TABLE test (id INTEGER NOT NULL PRIMARY KEY, value TEXT NOT NULL)"}, _threadID);
```

Comment:
I think everything below here should go in the actual test, not `setup`.

Reply:
I see that is the way other tests are written, but wasn't sure about that. Do you have a reason why these prep steps don't get put in "setup"?

Reply:
Bump. Any reason that "prepping the test" doesn't go in "setup"? IMO there's no reason to have a "setup" function if the only thing that ever goes in there is starting the cluster.

Reply:
These test classes are set up to run multiple tests in a row. Some of these have tens of different tests in them. "Setup" is a place to put things that would be done before *all* of the tests, like bringing up a cluster. If logic is specific to a specific test, it should go in that test function.

Basically each test should be self-contained unless there's obvious shared logic between tests.

Reply:
Got it, that makes sense, thanks.